Doc/library/gzip.rst

```diff
@@ -222,25 +222,27 @@ Once executed the :mod:`gzip` module keeps the input file(s).
 .. versionchanged:: 3.8
 
    Add a new command line interface with a usage.
+   By default, when you will execute the CLI, the default compression level is 6,
+   it's a good tradeoff between the best and fast compression methods.
```

Comment:
I suggest to remove "it's a good tradeoff between the best and fast compression methods". I wouldn't promise that it's a *good* tradeoff. Compression is full of bad surprises.

Just one example from gzip manual page:
"In  some rare cases, the --best option gives worse compression than the default compression level (-6). On some highly  redundant  files,  compress compresses better than gzip."